src/Microsoft.SqlTools.ManagedBatchParser/ReliableConnection/ReliableConnectionHelper.cs

```diff
@@ -776,8 +776,9 @@ public static ServerSystemInfo GetServerSystemInfo(IDbConnection connection)
             }
             catch (Exception ex)
             {
+                 // we don't want to fail when an error occurs while fetching these properties.
```

Comment:
Would be good to log WHY we don't want to fail here. (something along the lines of "these properties may not be available for all server types or users and since these properties aren't critical it's ok to not always include them). 

You should also make these properties optional in the typings file on the ADS side so callers know to handle them not being there. (this is assuming that these values default to null, which I think should be how they're handled so callers know whether they weren't retrieved or actually came back as 0 for some reason)